mscore/inspector/inspectorFret.cpp

```diff
@@ -36,16 +36,32 @@ InspectorFretDiagram::InspectorFretDiagram(QWidget* parent)
             { Pid::PLACEMENT,    0, f.placement,   f.resetPlacement   },
             { Pid::FRET_STRINGS, 0, f.strings,     f.resetStrings     },
             { Pid::FRET_FRETS,   0, f.frets,       f.resetFrets       },
-            { Pid::FRET_BARRE,   0, f.barre,       f.resetBarre       },
+            { Pid::FRET_NUT,     0, f.showNut,     f.resetShowNut     },
             };
       const std::vector<InspectorPanel> ppList = {
             { f.title, f.panel }
             };
+
+      dotTypeButtons = {
+            f.circleSelect,
+            f.crossSelect,
+            f.triangleSelect,
+            f.squareSelect
+            };
+
       mapSignals(iiList, ppList);
       int fretNumber = toFretDiagram(inspector->element())->fretOffset() + 1;
       f.fretNumber->setValue(fretNumber);
```

Comment:
If you are still working on this, would you consider adding
```
      f.resetFretNumber->setEnabled(fretNumber != 1);
```
here? I included it in `fretNumberChanged()`, but I missed putting it in the constructor.

Reply:
Sure, will do.